tests/utils.js

```diff
@@ -222,7 +222,7 @@ const deleteAll = (except = []) => {
             console.log(`Deleted docs: ${JSON.stringify(response)}`);
           }
         }),
-        sentinel.allDocs({keys: infoIds})
+        module.exports.sentinelDb.allDocs({keys: infoIds})
```

Comment:
This and the below test changes are broken in master, but we haven't noticed because folks have CouchDB running all the time (so these "unit" tests are actually hitting CouchDB)

Reply:
:o